index.html

```diff
@@ -0,0 +1,164 @@
+        </div> 
+
+
+        <div id="heading-cont" class="heading">
+            <h2 id="main-subtitle" class="main-subtitle">International Culinary Summit</h2>
+            <h1 id="main-title" class="main-title">FLAVORS OF PERU 2022</h1>
+        </div>
+
+        <div id="description-cont">
+            <p id="description" class="description">A closer look at the evolution of Peruvian cuisine, in hand with 5 of the country's most laureated representatives. The ultimate celebration of the senses.</p>
+        </div>
+
+        <div id="dates-venue">
+            <h2 id="dates" class="dates">2022/07/27(WED) - 29(FRI)</h2>
+            <h4 id="venue" class="venue">@ Admiralspalast Berlin, DE</h4>
+        </div>
+
+    </header>
+
+    <section id="program" class="program-container">
+
+        <h2 id="program-main">Main Program</h2>
+
+        <hr class="separator">
+
+        <div id="program-cards-container">
+            <div  class="program-card">
+                <img class="card-icon" src="images/icons/keynote.png" alt="">
+                <p class="card-title">Lecture</p>
+                <p class="card-description">Listen to the event's keynote speakers about their culinary industry journey and learn from their experiences.</p>
+            </div>
+
+            <div class="program-card">
+                <img class="card-icon" src="images/icons/dish.png" alt="">
+                <p class="card-title">Tasting</p>
+                <p class="card-description">Enjoy yourself with an exquisite showcase of Peru's most dishes and beverages.</p>
+            </div>
+
+            <div class="program-card">
+                <img class="card-icon" src="images/icons/chat.png" alt="">
+                <p class="card-title">Forum</p>
+                <p class="card-description">Have the chance to share your thoughs, questions and experiences not only with the speakers of the event, but with colleagues from all around the world.</p>
+            </div>
+
+            <div class="program-card">
+                <img class="card-icon" src="images/icons/chef.png" alt="">
+                <p class="card-title">Workshop</p>
+                <p class="card-description">Be part of exclusive workshops and learn how to use peruvian ingredients in different local cuisines.</p>
+            </div>
+    
+            <div class="program-card">
+                <img class="card-icon" src="images/icons/theatre.png" alt="">
+                <p class="card-title">Shows</p>
+                <p class="card-description">Be part of the celebration of peruvian culture. Assist to dance shows, concerts, art showcases and surprises.</p>
+            </div>
+        </div>
+
+        <button id="submit-button">Save your spot here</button>
+        
+    </section>
+
+
+
+    <section id="speakers">
+
+        <h2 id="speakers-title">Featured speakers</h2>
+        <hr class="separator" id="speakers-separator">
+
+        <span id="speakers-container"></span>
```

Comment:
- [ ] I think the chess table for each image could improve on the position regarding the image and color, please fix it to be more according to the given design. 🐻 
- Template:
![image](https://user-images.githubusercontent.com/20563252/137224979-e343bc11-90a0-4b6a-be6e-2d42901705a4.png)
- Your design:
![image](https://user-images.githubusercontent.com/20563252/137224995-75b9edc3-8af3-4ffb-ae75-3f258a088fc1.png)